src/app/components/pie-chart/pie-chart.css

```
@@ -22,6 +24,42 @@ mat-card {
   margin: 10px 0;
 }
 
-.legend-labels {
-  width: auto !important;
+::ng-deep .chart-legend {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)